src/index.html

```diff
@@ -41,6 +42,10 @@ <h2>Add your Score</h2>
       </form>
     </section>
   </main>
+
+  <footer>
+    <!-- <img src="./images/bg-3.jpg" alt="Image of a beautiful view"> -->
```

Comment:
[OPTIONAL] It would be better to remove any unused code to keep your code clean.